test is failing:

```
Error: [ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.875 s <<< FAILURE! -- in org.sakaiproject.assignment.impl.AssignmentTransferCopyEntitiesTest
Error: [ERROR] org.sakaiproject.assignment.impl.AssignmentTransferCopyEntitiesTest.testTransferCopyEntitiesWithTagIntegration -- Time elapsed: 0.082 s <<< FAILURE!
Wanted but not invoked:
tagService.getAssociatedTagsForItem(
    "f5dae634-fb45-47ed-9cca-9ce91217c6f4",
    "997f9eef-9602-4c1e-b68e-13c08abb2595"
);
-> at org.sakaiproject.assignment.impl.AssignmentTransferCopyEntitiesTest.testTransferCopyEntitiesWithTagIntegration(AssignmentTransferCopyEntitiesTest.java:460)
Actually, there were zero interactions with this mock.

	at org.sakaiproject.assignment.impl.AssignmentTransferCopyEntitiesTest.testTransferCopyEntitiesWithTagIntegration(AssignmentTransferCopyEntitiesTest.java:460)
```

--- a/assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentTransferCopyEntitiesTest.java
+++ b/assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentTransferCopyEntitiesTest.java
@@ -72,6 +72,7 @@ import org.sakaiproject.grading.api.GradebookInformation;
 import org.sakaiproject.grading.api.GradingConstants;
 import org.sakaiproject.grading.api.GradingService;
 import org.sakaiproject.site.api.SiteService;
+import org.sakaiproject.taggable.api.TaggingManager;
 import org.sakaiproject.tags.api.Tag;
 import org.sakaiproject.tags.api.TagService;
 import org.sakaiproject.time.api.UserTimeService;
@@ -412,7 +413,7 @@ public class AssignmentTransferCopyEntitiesTest extends AbstractTransactionalJUn
 
     @Test
     public void testTransferCopyEntitiesWithTagIntegration() throws Exception {
-        when(serverConfigurationService.getBoolean("tagservice.enable.integrations", false)).thenReturn(true);
+        when(serverConfigurationService.getBoolean("tagservice.enable.integrations", true)).thenReturn(true);
 
         String fromContext = UUID.randomUUID().toString();
         String toContext = UUID.randomUUID().toString();

